src/conductor/get-third-party.ts

```diff
@@ -1,6 +1,7 @@
 import pkgUp from 'pkg-up';
+
+import { parseJsonFile } from '../helpers';
 import fs from 'fs';
```

Comment:
I would also try to use the same pattern of ordered imports - like fs needs to come at the beginning